cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
Actually, I can get a few `OK`s if I wait a very long time, but I get `NOACK` most of the time.

> The first thing you could check is if #1703 works on Firefly motes, e.g., by running examples/llsec/ipv6-demos/simple-udp-rpl. Since I only have OpenMotes, I can not try this on my own.

It does not work. See https://github.com/contiki-os/contiki/pull/1703#issuecomment-225736642.

> Furthermore, if you enabled the debug output in potr.c and secmac.c, I might be able to identify the problem.

See `sender.log` and `receiver.log` there:
https://gist.github.com/bthebaudeau/1d8d9669c3b8ca40aaf0133a849d0fe6

The frequency of the `OK`s seems to be a little bit higher with `DEBUG`, but it's still very far from working smoothly. The nodes are only 1 cm apart.